hw06-atm-emulator/src/main/java/ru/otus/Builders/IDispenserBuilder.java

```diff
@@ -0,0 +1,7 @@
+package ru.otus.Builders;
+
+import ru.otus.IDispenser;
+
+public interface IDispenserBuilder {
```

Comment:
В Java не принято имена интерфейсов начинать с "I"
```suggestion
public interface DispenserBuilder {
```